CharacterCreation/01_Races.txt

```diff
@@ -226,7 +227,7 @@ Seadevil
 -1 to base CHA, -1 to base FOR.
 +1 to DEX
 Low-light vision.
-Ability: Bioluminesce - Once per day.[Is an available skill] Persuade a being, sentient or not, that you are trustworthy. Victim must roll a will-save 50+(Siren points). Effect is dissipated when victims eyes are covered or lose sight of you. This takes a tremendous amount of effort and you take a -10 penalty if tired or bloodied. This ability will temporarily scramble a robot’s visual cortex, but they can adjust rapidly.
+Ability: Bioluminescence - Once per day.[Is an available skill] Persuade a being, sentient or not, that you are trustworthy. Victim must roll a will-save 50+(Siren points). Effect is dissipated when victims eyes are covered or lose sight of you. This takes a tremendous amount of effort and you take a -10 penalty if tired or bloodied. This ability will temporarily scramble a robot’s visual cortex, but they can adjust rapidly.
 
 Chondrich
 -1 to base CHA, +1 to both STR and FOR

```

Comment:
also here, for the "FOR" -> "FORT" change.

Reply:
Can Do, Will Do! :)